src/commands/index.rs

```diff
@@ -67,6 +67,13 @@ fn add_field(index_name: &str, field_name: &str, path: &str) -> RedisResult {
     }
 }
 
+fn del_index(index_name: &str) -> RedisResult {
```

Comment:
Let's be consistent with the naming, and call the function `del_schema`. Way back we used the 'schema' term in this modules since the name 'index' was too overloaded.

Reply:
done